Makefile

```diff
@@ -154,6 +156,17 @@ JEKYLL_FLAGS   =
 
 BUILD_DATA = $(DOCS_VERSION_DATA) $(DOCS_PAGE_LIST) $(TOC_FILES)
 
+ifdef WINDOWS
+	VERSION_CONFIG       := $(subst /,\,$(VERSION_CONFIG))
+	DEFAULTS_CONFIG      := $(subst /,\,$(DEFAULTS_CONFIG))
+	DOCS_PAGE_LIST       := $(subst /,\,$(DOCS_PAGE_LIST))
+	DOCS_VERSION_DATA    := $(subst /,\,$(DOCS_VERSION_DATA))
+	TOC_FILES            := $(subst /,\,$(TOC_FILES))
+	PLUGINS_APP          := $(subst /,\,$(PLUGINS_APP))
+	CSS_DEST_DIR         := $(subst /,\,$(CSS_DEST_DIR))
+	FETCHED_FILES        := $(subst /,\,$(FETCHED_FILES))
+endif
```

Comment:
This could break other uses of these files. Instead, change the definition of `RM` and `RMDIR` to escape slashes, like how `makedir` does it [here](https://github.com/apache/cordova-docs/blob/master/Makefile#L36).